docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+  #:ref "1474d32"
+  #:loc "98"
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "ctc-getinfo"]{/ctc/getInfo}
+
+The @urlin{/ctc/getInfo} route corresponds to a @jsin{getInfo} operation on
+@tech{contracts} when using the standard library directly.
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+                                     ------------------
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+                                        ------------------------------
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "forget-acc"]{/forget/acc}
+See: @secref{id-clean-up}.
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "forget-ctc"]{/forget/ctc}
+See: @secref{id-clean-up}.
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "backend"]{/backend/*}
+
+@(define link-backend-stdlib @reachexlink[
+  #:ref "27fb2a9"
+  #:loc (cons "41" "52")
+  "tut-6/index.mjs"
+  "this code from the (non-RPC) \"tut-6\""])
+
+@(define link-backend-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc (cons "122" "144")
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+The @urlin{/backend} namespace is dynamically populated with routes which are
+specific to your @|DApp|, e.g. @urlin{/backend/Alice}, and
+@urlin{/backend/Bob}, or @urlin{/backend/SomeOtherParticipant} whose logic
+you've encoded.
+
+Contrast @link-backend-stdlib example:
+@js{
+  await Promise.all([
+    backend.Alice(ctcAlice, {
+      ...Player('Alice'),
+      wager: stdlib.parseCurrency(5),
+    }),
+    backend.Bob(ctcBob, {
+      ...Player('Bob'),
+      acceptWager: (amt) => {
+        console.log(`Bob accepts the wager of ${fmt(amt)}.`);
+      },
+    }),
+  ]);
+}
+
+...with the equivalent @link-backend-rpc Python example:
+@py{
+  def play_alice():
+      rpc_callbacks(
+          '/backend/Alice',
+          ctc_alice,
+          dict(wager=rpc('/stdlib/parseCurrency', 5), **player('Alice')))
+
+  def play_bob():
+      def acceptWager(amt):
+          print('Bob accepts the wager of %s' % fmt(amt))
+
+      rpc_callbacks(
+          '/backend/Bob',
+          ctc_bob,
+          dict(acceptWager=acceptWager, **player('Bob')))
+
+  alice = Thread(target=play_alice)
```

Comment:
You shouldn't talk about threading when talking about the server API, but when you talk about the Python client, you should talk about threads fulfill a role similar to promises